src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -633,8 +633,16 @@ object Helpers extends SparkSessionWrapper {
                                  )
 
   def globPath(path: String, fromEpochMillis: Option[Long] = None, untilEpochMillis: Option[Long] = None): Array[PathStringFileStatus] = {
+    globPath(path, spark.sparkContext.hadoopConfiguration, fromEpochMillis, untilEpochMillis)
+  }
+
+  def globPath(path: String, conf: SerializableConfiguration, fromEpochMillis: Option[Long],
+               untilEpochMillis: Option[Long]): Array[PathStringFileStatus] = {
+    globPath(path, conf.value, fromEpochMillis, untilEpochMillis)
+  }
+
+  def globPath(path: String, conf: Configuration, fromEpochMillis: Option[Long], untilEpochMillis: Option[Long]): Array[PathStringFileStatus] = {
```

Comment:
Same [here](https://github.com/databrickslabs/overwatch/blob/993ea86e956e7e01453fc58de33e2543392ea957/src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala#L677) as it is also used inside the lamda. 

I like your code better, and I tried several ways but kept getting serializable issues. Could have been REPL related issues but just wanted to make sure you tested it.

Reply:
fixed